src/roles/wildchild.py

```diff
@@ -131,29 +119,27 @@ def on_chk_nightdone(evt, var):
 def on_transition_day_begin(evt, var):
     if (not var.START_WITH_DAY or not var.FIRST_DAY) and var.FIRST_NIGHT:
         for child in get_all_players(("wild child",)):
-            if child.nick not in IDOLS:
-                pl = get_players()
-                pl.remove(child)
-                if pl:
-                    target = random.choice(pl)
-                    IDOLS[child.nick] = target.nick
-                    child.send(messages["wild_child_random_idol"].format(target))
+            if child not in IDOLS:
+                players = get_players()
+                players.remove(child)
+                if players:
+                    random_idol(child, players)
```

Comment:
keep this inline. Don't split something out into a helper method unless it will actually be called from multiple places.

Reply:
I split it off because the indentation level was getting really deep and making it hard to read.